enterprise/cmd/executor/internal/run/run.go

```diff
@@ -64,8 +65,13 @@ func StandaloneRunRun(ctx context.Context, logger log.Logger, cfg *config.Config
 		if err != nil {
 			return err
 		}
-		if err := validateSrcCLIVersion(ctx, client, opts.QueueOptions.BaseClientOptions.EndpointOptions); err != nil {
-			return err
+		if err = validateSrcCLIVersion(ctx, client, opts.QueueOptions.BaseClientOptions.EndpointOptions); err != nil {
+			if errors.Is(err, ErrSrcPatchBehind) {
+				// This is ok. The patch just doesn't match but still works.
+				logger.Warn("Src patch version mismatch", log.Error(err))
```

Comment:
can we add some more context to what that means in the log msg?
```suggestion
				logger.Warn("A newer patch release version of src-cli is available, consider running executor install src-cli to upgrade", log.Error(err))
```

Reply:
the same in validate